repository-data/webfiles/src/main/resources/site/scss/components/_site-header.scss

```diff
@@ -1,46 +1,213 @@
 @import "../environment/tools/mixins/static-button";
+@import "../environment/tools/functions/easing";
 
-/* Site header */
-.site-header {
-    position: relative;
-    border-bottom: 1px solid $col-primary;
-    padding: round($gutter * 1.375) 0;
-    vertical-align: bottom;
+$burger-piece-width: 32px;
+$burger-piece-height: 3px;
+
+@mixin berger-menu(){
+    .list {
+        display: block;
+        margin-bottom: round($gutter * 0.5);
+
+        li {
+            display: block;
+            width: 100%;
+            margin-left: 0;
+            padding: $gutter 0;
+            border-top: 1px solid lighten($col-senary, 10%);
+
+            &:first-child {
+                border-top: none;
+            }
+        }
+    }
+}
 
-    .search-strip {
+@mixin berger-button(){
+    .main-nav__burger {
+        border: 3px solid transparent;
         position: absolute;
+        top: 5px;
         right: 0;
-        width: 50%;
+        z-index: 100;
+
+        @include media($min-width: $desktop-width, $ignore-for-ie: true) {
+            background-color: #005EB8;
+            border: 3px solid #005EB8;
+            padding: 3px 7px;
+            color: $white;
+
+            &:after {
+                content: "Menu";
+                margin-left: 5px;
+                font-size: 18px;
+                vertical-align: middle;
+            }
+        }
+
+        &:focus,
+        &:active,
+        &:hover {
+            border: 3px solid #FFB81C;
```

Comment:
I'd normally try to use `$yellow` here instead of hardcoding the colour

Reply:
Done